src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private double radius;
+
+    public void setRadius(double radius) {
+        if (radius <= 0) {
+            radius = 1;
+        }
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    @Override
+    public void setArea() {
```

Comment:
```suggestion
    public void calculateArea() {
```
we use setter when we insert value into field, for example public void setArea(double area)